<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTest1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>testi1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>testi2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lavelResult">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>tulos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Kerro nimesi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textName">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenPage2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>avaa sivu2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
